lametro/events.py

```diff
@@ -279,7 +85,7 @@ def scrape(self, window=None):
             )
         )
 
-        for event, web_event in self._merge_events(events):
+        for event, web_event in PairedEventStream(events, find_missing_partner=True):
```

Comment:
What a great abstraction, @hancush! This makes the scrape logic so much more understandable.